zipline-loader/src/androidMain/kotlin/app/cash/zipline/loader/internal/internalAndroid.kt

```diff
@@ -16,6 +16,11 @@
 package app.cash.zipline.loader.internal
 
 import app.cash.zipline.Zipline
+import kotlin.coroutines.CoroutineContext
+import kotlinx.coroutines.Dispatchers
 
-internal actual fun Zipline.multiplatformLoadJsModule(bytecode: ByteArray, id: String) =
+internal actual fun Zipline.multiplatformLoadJsModule(bytecode: ByteArray, id: String) {
   loadJsModule(bytecode, id)
+}
+
+actual val ioDispatcher: CoroutineContext = Dispatchers.IO
```

Comment:
I am _extremely_ apprehensive about having static dispatchers encoded rather than using inversion of control.